tests/src/test/resources/pom.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It is possible to rename this pom.xml to `bom-with-typo-artifact.xml`? (Does that still work?)